Dockerfile.autobuild

```diff
@@ -14,31 +14,45 @@
 # Base system is the LTS version of Ubuntu.
 FROM python:3.6-stretch
 
-RUN groupadd -g 2300 tmpgroup && usermod -g tmpgroup www-data && groupdel www-data && groupadd -g 1000 www-data && usermod -g www-data www-data && usermod -u 1000 www-data && groupdel tmpgroup
-
-RUN apt-get update
-RUN apt-get -y install nginx nginx-extras pwgen
+RUN groupadd -g 2300 tmpgroup \
+ && usermod -g tmpgroup www-data \
+ && groupdel www-data \
```

Comment:
What's the reason why you change the www-data's gid to 1000? 

If you want to use the same uid as your host machine, just use run_docker_with_current_user script : )

Feel free to tell me If you have any other concern.

Reply:
That's just trying to replicate the original logic.

Reply:
I'm happy to remove that part but it's not the concern of this pull request